core/tests/test_minimal_example.py

```diff
@@ -29,8 +30,10 @@ def tearDownClass(cls):
         cls.driver.quit()
 
     def setUp(self):
-        address = 'file://{}/webviz_example/index.html'.format(self.tempdir)
+        address = 'file://{}/webviz_example/sub_pages/markdown_example_2.html'\
+                  .format(self.tempdir)
         self.driver.get(address)
+        time.sleep(3)
```

Comment:
Why is there a need to sleep here?

Reply:
Do you have a suggestion for checking when mathjax has finished rendering (if started at all)?